websites/T/Tellingtone/presence.ts

```diff
@@ -0,0 +1,98 @@
+const presence = new Presence({ clientId: "1006201873985961984" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			viewHome: "general.viewHome",
+			paused: "general.paused",
+			playing: "general.playing",
+			episode: "general.episode",
+			viewPage: "general.viewPage",
+			buttonViewSeries: "general.buttonViewSeries",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+let strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null,
+	currentListening: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = document.location,
+		pathSplit = pathname.split("/"),
+		[newLang, time, showCover] = await Promise.all([
+			presence.getSetting<string>("lang").catch(() => "en"),
+			presence.getSetting<boolean>("time"),
+			presence.getSetting<boolean>("cover"),
+		]);
+
+	if (oldLang !== newLang || !strings) {
+		oldLang = newLang;
+		currentListening =
+			newLang === "fr" ? "Écoute actuellement " : "Currently listening to ";
```

Comment:
Is there no string for this? Otherwise it could be an option to make one

Reply:
I've searched in general localization an some other presence localization, there's "Listening to music" but not "currently listening to"

Reply:
I've just double check now, there's no string for this.

Reply:
@Bas950 what should I do?

Reply:
@RisingSunLight42 Make a PR on https://github.com/PreMiD/Localization edit the `src/presences/general.json` file and add a `Currently listening to` string.

Reply:
I'll do that when I can, thanks for the quick answer 🙏

Reply:
PR opened : https://github.com/PreMiD/Localization/pull/7210

Reply:
Please add the newly added string to the code section 👍🏼

Reply:
Of course 👍

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/6626/commits/4817bc9eddc30248860870d495319dff3e8a28d2 !
Thanks for your help